src/extension-test/integration/suite/jack-in-test.ts

```diff
@@ -79,7 +79,7 @@ async function startJackInProcedure(suite: string, cmdId: string, projectType: s
   testUtil.log(suite, 'test.clj opened');
 
   const projectRootPath = await projectRoot.findClosestProjectRootPath();
-  const projetcRootUri = vscode.Uri.file(projectRootPath);
+  const projetcRootUri = projectRootPath;
```

Comment:
Maybe I lack some context, but doesn't it seem like we could do it like so?

``` clojure
const projectRootUri = await projectRoot.findClosestProjectRootPath();
```

Reply:
I think you're right, I'll check & update.